### DISCUSSION TITLE
postcss-preset-mantine theme() override tailwind

### DISCUSSION THREAD

**Discussion by conduongtong** [mantinedev/mantine#6498] [Q&A] [answered]
I want to use tailwind theme function instead of mantine, how can I do that?
![image](https://github.com/mantinedev/mantine/assets/43648013/e00db5c6-afa5-450e-9465-1e3ab2ae0fb2)

mantine preset: https://github.com/mantinedev/postcss-preset-mantine/blob/master/src/preset.ts
tailwind: https://tailwindcss.com/docs/functions-and-directives#theme

**Comment by rtivital** [comment, marked as answer]
To fix the issue, update `postcss-preset-mantine` to version 1.16.0.

**Reply by conduongtong** [reply]
Thank youuuu. Appreciated
